packages/website/src/content/docs/setup/laravel.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Using Spotlight with Laravel
+description: A guide on how to setup Spotlight for Laravel
+---
+
+import { Content as InstallCommand } from "../../../components/InstallCommand.mdx";
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+This guide gets Spotlight up and running in Laravel with Sentry SDK.
+
+## 1. Installation
+
+### Install Sentry SDK
+
+Follow the [Laravel Getting Started](https://docs.sentry.io/platforms/php/guides/laravel/#install) guide to install and configure Sentry SDK for Laravel.
+
+Note that you don't need a DSN or don't need to sign up for Sentry to use Spotlight. If you already have Sentry SDK installed, make sure you have a recent version.
+
+### Get and Run Spotlight
+
+<Tabs>
+  <TabItem label="npx">```shell npx @spotlightjs/spotlight ```</TabItem>
+  <TabItem label="docker">
+    ```shell docker run --rm -p 8969 ghcr.io/getsentry/spotlight:latest ```
+  </TabItem>
+  <TabItem label="binary">
+    ```shell curl -q https://spotlightjs.com/install.sh | sh ```
+  </TabItem>
+</Tabs>
+
+## 2. Enable Spotlight
+
+### Configure Sentry
+
+In your `config/sentry.php` file, uncomment or add the spotlight configuration:
+
+```php
+'spotlight' => env('SENTRY_SPOTLIGHT', false),
```

Comment:
I think this is the default now so we should probably remove this section: https://github.com/getsentry/sentry-php/pull/1789

Reply:
Not in laravel https://github.com/getsentry/sentry-laravel/blob/7dda008a8812f7b3747499ec85eefac04de55dfa/config/sentry.php#L14
for whatever reason 🤷‍♂️